runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -373,18 +377,47 @@ void requestSync()
 	}
 
 	@Subscribe
-	public void onCharacterNameUpdate(final CharacterNameUpdate event)
+	public void onStatusUpdate(final StatusUpdate event)
```

Comment:
This should probably drop status updates for self since it will race if multiple updates are sent out before one is received